src/modules/navigator/mission_block.cpp

```diff
@@ -277,9 +277,15 @@ MissionBlock::is_mission_item_reached()
 			_time_wp_reached = now;
 
 		} else {
-			/*normal mission items */
 
-			float mission_acceptance_radius = _navigator->get_acceptance_radius();
+			float acceptance_radius = _navigator->get_acceptance_radius();
+
+			// We use the acceptance radius of the mission item if it has been set (not NAN)
+			// but only for multicopter.
+			if (_navigator->get_vstatus()->vehicle_type == vehicle_status_s::VEHICLE_TYPE_ROTARY_WING
```

Comment:
Can't this logic live inside of `get_acceptance_radius()`? We could again pass `_mission_item.acceptance_radius` as an argument.

Reply:
I actually found the way it was before quite confusing. So `get_acceptance_radius()` had this overload but the overload was only used from this location, right?

Reply:
Right. If one anyway only wants to take the acceptance radius from the param then using get_default_acceptance_radius() would be cleaner though no? And always pass `_mission_item.acceptance_radius` as argument of `get_acceptance_radius()`?